design-library-usage-sample/src/main/java/com/what3words/design/library/sample/ui/screens/IconButtonScreen.kt

```diff
@@ -0,0 +1,132 @@
+package com.what3words.design.library.sample.ui.screens
+
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.rememberScrollState
+import androidx.compose.foundation.verticalScroll
+import androidx.compose.material.icons.Icons
+import androidx.compose.material.icons.filled.Add
+import androidx.compose.material3.FilledIconButton
+import androidx.compose.material3.FilledTonalIconButton
+import androidx.compose.material3.Icon
+import androidx.compose.material3.IconButton
+import androidx.compose.material3.MaterialTheme
+import androidx.compose.material3.OutlinedIconButton
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.unit.dp
+import com.what3words.design.library.ui.theme.W3WTheme
+
+/**
+ * A screen showcasing various styles of icon buttons in Material Design styled by both [MaterialTheme] and [W3WTheme].
+ * This screen displays different types of icon buttons including standard, filled,
+ * tonal, and outlined. Each button is presented in both its enabled and disabled states
+ * to demonstrate the visual changes in different states.
+ *
+ * This screen is useful for designers and developers to understand the styling options
+ * and states of icon buttons in a Material Design-based UI.
+ */
+@Composable
+fun IconButtonScreen() {
```

Comment:
In adherence to Jetpack Compose API guidelines, it is advised that every composable function should embrace and adhere to the inclusion of a Modifier-type parameter.

The guidelines state, "_Element functions MUST accept a parameter of type Modifier. This parameter MUST be named “modifier” and MUST appear as the first optional parameter in the element function's parameter list. Element functions MUST NOT accept multiple Modifier parameters._" Further details can be found [here](https://android.googlesource.com/platform/frameworks/support/+/androidx-main/compose/docs/compose-api-guidelines.md#elements-accept-and-respect-a-modifier-parameter:~:text=Elements%20accept%20and,multiple%20Modifier%20parameters.)